mappings/net/minecraft/world/World.mapping

```diff
@@ -21,12 +21,21 @@ CLASS net/minecraft/class_18 net/minecraft/world/World
 		ARG 4 seed
 	METHOD method_149 getClosestPlayer (DDDD)Lnet/minecraft/class_54;
 	METHOD method_150 playSound (DDDLjava/lang/String;FF)V
+	METHOD method_155 isRegionLoaded (IIIIII)Z
 	METHOD method_156 (IIIIZI)Z
 		ARG 6 side
 	METHOD method_159 setTime (J)V
 		ARG 1 time
 	METHOD method_168 canSpawnEntity (Lnet/minecraft/class_25;)Z
 		ARG 1 box
+	METHOD method_169 isBoxInMaterial (Lnet/minecraft/class_25;Lnet/minecraft/class_15;)Z
+		ARG 1 boundingBox
+		ARG 2 material
+	METHOD method_170 updateMovementInFluid (Lnet/minecraft/class_25;Lnet/minecraft/class_15;Lnet/minecraft/class_57;)Z
```

Comment:
[updateMovementInFluid](https://maven.fabricmc.net/docs/yarn-1.20-pre4+build.2/net/minecraft/entity/Entity.html#updateMovementInFluid(net.minecraft.registry.tag.TagKey,double) is the yarn name, since its in the entity class it would be Entity.updateEntityMovementInFluid

Reply:
Ok makes sense, keep it as is